curriculum/challenges/ukrainian/05-back-end-development-and-apis/back-end-development-and-apis-projects/timestamp-microservice.md

```diff
@@ -1,26 +1,26 @@
 ---
 id: bd7158d8c443edefaeb5bdef
-title: Timestamp Microservice
+title: Мікросервіс часової мітки
 challengeType: 4
 forumTopicId: 301508
 dashedName: timestamp-microservice
 ---
 
 # --description--
 
-Build a full stack JavaScript app that is functionally similar to this: <https://timestamp-microservice.freecodecamp.rocks/>. Working on this project will involve you writing your code using one of the following methods:
+Створіть full stack додаток на JavaScript, який функціонально схожий до цього: <https://timestamp-microservice.freecodecamp.rocks/>. Робота над цим проектом залучатиме тебе писати свій код використовуючи один з наступних методів:
```

Comment:
This link looks malformed

Reply:
The link is unchanged - this is how it appears on English.

It's wrapped in `<>` so that we can render the raw link directly, while keeping the linter from being angry about "no-bare-urls".